CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
 # Cromwell Change Log
 
 ## 0.20
+
+* The default per-upload bytes size for GCS is now the minumum 256K
+instead of 64M. There is also an undocumented config key
```

Comment:
At the moment we weren't sure how to handle this or what the ideal default would be. Putting it in the config allows gotc to fiddle with it easily. I'm guessing undocumented so that we can remove it later if it makes sense do handle this some other way